### ISSUE TITLE
PDF is Missing Text

### ISSUE THREAD

**Issue by akozmic** [mozilla/pdf.js#7450]
Link to PDF file:  https://www.dropbox.com/s/3hmtnbyxyuz3cwr/Contract%20902%205-1.pdf?dl=0

Configuration:
- Browser and its version:  Google Chrome Version 50.0.2661.102 (64-bit)
- Operating system and its version:  OSX 10.9.5 
- PDF.js version:  Latest (1.5.188)
- Is an extension: No

Steps to reproduce the problem:
1. Open PDF in Adobe Reader or Google Chrome - notice that Buyer(s), Seller(s), Property PIN, Property Address, etc all have Text next to them.  These are _not_ traditional Form Fields. 
2. Open PDF in Demo Viewer - https://mozilla.github.io/pdf.js/web/viewer.html,  notice that none of the text from step 1 is there. 

What is the expected behavior? (add screenshot)
https://www.dropbox.com/s/72oktg0sfmzzub9/Screen%20Shot%202016-06-28%20at%2010.41.55%20AM.png?dl=0

What went wrong? (add screenshot)
https://www.dropbox.com/s/gk6dnapie0pztgp/Screen%20Shot%202016-06-28%20at%2010.43.06%20AM.png?dl=0

**Comment by THausherr** [comment]
Nicole can be found here:
Root/Pages/Kids/[0]/Resources/XObject/X6/Resources/XObject/Annot_333657395

The problem is that in the content stream, it names the font "/Helvetica". That one doesn't exist. What does exist is "Helv".

In PDFBox "/Helvetica 8 Tf" does nothing, but when handling Tj a default font is used.

The same problem can be found here:
https://issues.apache.org/jira/secure/attachment/12551480/forms26916.pdf
The second phone line ("https://issues.apache.org/jira/secure/attachment/12551480/forms26916.pdf") is missing.

**Comment by timvandermeij** [comment]
How do other PDF readers handle such a situation, or is this PDF file broken in other viewers too?

**Comment by THausherr** [comment]
Adobe Reader and GSView display the missing text.

**Comment by akozmic** [comment]
Any updates on this ticket? Thanks!

**Comment by linux40** [comment]
This may cased by the lack of fonts. I use ghostscript to replace them. The following command works for me but I don't know whether and how to customize the fonts.
```
gs -sDEVICE=pdfwrite -dBATCH -dNOPAUSE -dLOCALFONTS -sOutputFile=output-file.pdf your-file.pdf
```

**Comment by Snuffleupagus** [comment]
WFM, with PDF.js version `2.7.510` in Firefox Nightly 87; most likely fixed by PR #11218.
